ProjectStructureAndRoles.md

```diff
@@ -71,9 +71,13 @@ The API onboarding and management in CAMARA is described in [API-onboarding.md](
 
 **Commonalities**
 
-CAMARA commonalities shall be processed in a Working Group “Commonalities”. The Working Group “Commonalities” works under the supervision of the TSC. The scope of the Working Group is to create guidelines, best practices and concept documents relevant for all APIs in CAMARA. The commonalities Working Group shall work e.g. on the development of a common meta model that shall be used as a reference across the Sub Projects. It is open for members of any Sub Project or Working Group to raise issues in the Technical Steering Committee (TSC) for relevant topics on a need basis.
+CAMARA commonalities shall be processed in a Working Group "Commonalities”. The Working Group “Commonalities” works under the supervision of the TSC. The scope of the Working Group is to create guidelines, best practices and concept documents relevant for all APIs in CAMARA. The Commonalities Working Group shall work e.g. on the development of a common meta model that shall be used as a reference across the Sub Projects. It is open for members of any Sub Project or Working Group to raise issues in the Technical Steering Committee (TSC) for relevant topics on a need basis.
 
-All Sub Projects must comply with the work in the commonalities Working Group.
+All Sub Projects must comply with the work in the Commonalities Working Group.
```

Comment:
I don't believe there is any obligation to comply with updates that are not yet part of a release
```suggestion
All Sub Projects must comply with the latest release of the Commonalities Working Group guidelines.
```

Reply:
Somehow this PR pivoted in scope from adding ICM (which is not yet done) to correcting the Commonalities section. But as we will get the same sentence also for ICM it makes sense to discuss.

@eric-murray I don't like to reduce it just to "latest releases". During the release cycle there are for examples also pre-releases relevant which are excluded by the term "latest release" (at least in the sense GitHub is using it). 

How about just changing "work" into "work results"?

Reply:
Maybe we can review the text after the meta-release cycle is fully established and sub-project responsibilities clear.

I think sub-projects need clearer guidance on the specific Commonalities (and ICM) rules and guidelines they need to comply with at a given time. It is not unknown for changes to be made to the WIP documentation, only for those changes to be later modified before being "frozen" in a release. It is hard for sub-projects to try and hit a moving target.

I include pre-releases (release candidates) here, as I don't think it should be mandatory to comply even with a release candidate, only the release itself, though I understand the timescales for the current meta-release force us to do this at the moment.

Reply:
Yes, would be good to have this discussion independently. I agree that the current sentence is vague, but would like to keep it for now.